src/ott/experimental/mmsinkhorn.py

```diff
@@ -303,7 +303,7 @@ def __call__(
 
     cost_t = cost_tensor(x_s, cost_fns)
     state = self.init_state(n_s)
-    epsilon = 0.05 * jnp.mean(cost_t) if epsilon is None else epsilon
+    epsilon = 0.05 * jnp.std(cost_t) if epsilon is None else epsilon
```

Comment:
Maybe let's mention it in the docs how the default is set?

Reply:
I have made a pass in various places to ensure this is refreshed.